<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecurityWidget</class>
 <widget class="QWidget" name="SecurityWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Touch">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>301</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>启用触控权限策略</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QGroupBox" name="groupBox_TouchByGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>受权限控制</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <widget class="QListWidget" name="listT">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>所有组</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_TouchByBit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>受位控制</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>地址类型：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_addrTypeT">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>位地址</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>字地址</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>字的位：</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spin_bitIndexT">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>53</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>解锁条件：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_conditionT">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>为1时解锁</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>为0时解锁</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>地址：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_ctlAddrT">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>LB0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ctlAddrT">
     <property name="geometry">
      <rect>
       <x>201</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resourse.qrc">
       <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GroupV">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>启用显现权限策略</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QGroupBox" name="groupBox_ShowByGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>受权限控制</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <widget class="QListWidget" name="listV">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>所有组</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_ShowByBit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>受位控制</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="edit_ctlAddrV">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>LB0</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spin_bitIndexV">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_conditionV">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>为1时显现</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>为0时显现</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>地址：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>显现条件：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>地址类型：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_addrTypeV">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>位地址</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>字地址</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>字的位：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ctlAddrV">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resourse.qrc">
       <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Notice">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>295</y>
     <width>301</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>操作通知</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>53</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>地址：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_noticeAddr">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LB0</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_NoticceAddrType">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>位地址</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>字地址</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>地址类型：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>通知值：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_NoticeType">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>16位整数</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32位整数</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16位正整数</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32位正整数</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32位浮点数</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="edit_Value">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferNumbers</set>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_noticeAddr">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../../resourse.qrc">
      <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据类型：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group_press">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>按压延时</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>按压时间：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_time">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100毫秒</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
